introduction.md

```diff
@@ -0,0 +1 @@
+Hello. My name is James Shaeffer and I am taking the Mastering GitHub course at CodeSchool.com
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.